### DISCUSSION TITLE
How to add a nested table inside a Relation Manager Edit Modal

### DISCUSSION THREAD

**Discussion by ghost** [filamentphp/filament#3556] [Help]
A has many B
B has many C

Hello, in the edit page of a resource A that has a RelationManager, there is a table for the sub-relation B. When clicking the edit of any B row, a modal is shown; how can I show a table of C rows in the B modal?

Thank you

**Comment by danharrin** [comment]
You cant embed relation managers in each other, but you can use a repeater

https://filamentphp.com/docs/2.x/forms/fields#populating-automatically-from-a-relationship-2
